### ISSUE TITLE
[BUG][NOWEB] - interface delay /api/session/chats/xxx@c.us/messages

### ISSUE THREAD

**Issue by HuangDaHui** [allburov/whatsapp-http-api#461]
/api/TCN013/chats/8615018078614%40c.us/messages?downloadMedia=false&limit=100
I noticed that the interface seems to have a certain delay? 
When this interface cannot get the corresponding message, the reply function cannot be used normally.

**Comment by devlikepro** [comment]
> I noticed that the interface seems to have a certain delay? 

Hi! What do you mean by delay? 
like a timeout or when you get a webhook about new message the endpoint doesn't have the message yet?


[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by HuangDaHui** [comment]
Yes, the webhook has received the notification, but the query interface cannot find the relevant records.

**Comment by HuangDaHui** [comment]
> > 我注意到界面好像有一定的延迟？
> 
> 你好！你说的延迟是什么意思？ 比如超时，或者当你收到有关新消息的 webhook 时，端点还没有收到该消息？
> 
> [![赞助人：PRO](https://camo.githubusercontent.com/53c8c17e54974d30f0938bed9c2aa4e5c7dc7bc776296a6776840124e4086a04/68747470733a2f2f696d672e736869656c64732e696f2f62616467652f706174726f6e2d50524f2d313838613432)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

Yes, the webhook has received the notification, but the query interface cannot find the relevant records.
